### PR TITLE
Included two new files: card.html for the card view, details.html for…

### PR DESCRIPTION
**Pull request by jgiardino**
… the page that displays when clicking on a record in the list view or card view. Also updated the View icons, including title text as labels.

According to the HTML5 Specification, some elements cannot be nested inside an anchor `<a>` element. Some of the elements that can be included in the card or list view are on this list. For more details, view this post: http://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

**Updates**

Both views include updates to the view icons, based on chat with Leslie, to test if these are the icons we want. 

The list view includes the following updates:
1. The first list item includes the `<a>` inside div.list-group-item. Div.list-group-item also includes additional styles needed to implement this solution in the `<head>`. These styles enable the <a> to fill the div, but also display behind any clickable elements (e.g. checkbox, other links, actions)
2. The second list item is where I tested what happens when we simply change `<div>` to `<a>` and remove any of the forbidden elements. It's not pretty.
3. The third list item is where I changed `<div>` to `<a>`, with no other changes. This is also not pretty.

The card view includes the following updates:
1. The first list item includes the `<a>` inside div.card-pf. Div.card-pf also includes additional styles (read item #1 above)
2. The second list item is where I changed `<div>` to `<a>`, with no other changes. 

PLEASE NOTE I was missing the cards.less file in my build (???) so it's possible there are additional issues that will surface when that's included. I did see some minor layout issues on this page but am assuming those will be resolved when this file is finally included.

### CODE REVIEW

**Comment by jgiardino** [comment]
A few more comments...

I included all css in the `<head>` of the html files I updated. I expect this css would be refactored/moved to the correct css files if we decide we do want keep it.

On the details.html page, I pasted in the breadcrumb pattern, and just randomly linked one of the links to the card view and one to the list view. Obviously that would not be the case, but I did wonder what the expected behavior is when the user uses a breadcrumb. Is it expected to remember the selected view, or do we just display the default view?

**Comment by andresgalante** [comment]
@jgiardino this is awesome! This must be the best PR description I've ever read.

You are absolutely right about the nested anchors. What do you think if we go with a less hacky solution with a js event?

Also, this branch is not forked. @priley86 can you show Jenn how to do a fork please?

thanks a lot, Great PR!

File: dist/card.html [hunk 1]
```diff
@@ -0,0 +1,653 @@
...
+    <link rel="stylesheet" href="css/patternfly.min.css">
+    <link rel="stylesheet" href="css/patternfly-additions.min.css">
+    <style>
+      /*JENN - included styles to enable drilldown behavior to a contents/details page; only the first record demonstrates this behavior  */
+      .card-pf-view {display:block;}
+      .card-pf-view.jenn-test {pointer-events:none;}
+      .card-pf-view.jenn-test a, .card-pf-view.jenn-test input, .card-pf-view.jenn-test button {pointer-events: auto; z-index:0; position:relative;}
+
+      /*JENN - included table icon (created new class for table icon)  */
+      .fa-th-table:before {content: "\f0ce";}
```
Comment (andresgalante) [inline dist/card.html]: @jgiardino You can change this directly on the html: `<i class="fa fa-table" aria-hidden="true"></i>`

**Comment by jgiardino** [comment]
Hi Andres,
Thanks for reviewing! I'll make updates to use a JS event instead of adding the `<a>`. I assume I would continue working in patternfly-demo-app for creating updates to JS, but let me know if that's not the case. (I'm still a little fuzzy on the process for getting css/js updates into the core patternfly repo.)  I'll work with Patrick on creating a fork.

Thanks!

**Comment by andresgalante** [comment]
Thanks Jenn. Core PatternFly is another beast. The demo app is an implementation of patternfly. @priley86 can show you what is the best way to include js there.

Think about the demo app as a starting point for a developer that wants to build something with patternfly and lets try to keep it as much production ready as possible.

**Pull request closed without merging**